src/query/storage/local/config.go

```diff
@@ -66,10 +69,23 @@ func (c ClusterStaticConfiguration) newClient(
 // ClusterStaticNamespaceConfiguration describes the namespaces in a
 // static cluster.
 type ClusterStaticNamespaceConfiguration struct {
-	Namespace          string              `yaml:"namespace"`
-	StorageMetricsType storage.MetricsType `yaml:"storageMetricsType"`
-	Retention          time.Duration       `yaml:"retention" validate:"nonzero"`
-	Resolution         time.Duration       `yaml:"resolution" validate:"min=0"`
+	Namespace  string                                         `yaml:"namespace"`
+	Type       storage.MetricsType                            `yaml:"type"`
```

Comment:
re: our conversation about deprecated, looks like there is a convention for this in Go already: https://go-review.googlesource.com/c/blog/+/18956/2/content/godoc-documenting-go-code.article